src/Diagrams/Backend/SVG.hs

```diff
@@ -300,6 +299,10 @@ svgClass = curry keyVal "class"
 svgTitle :: SVGFloat n => String -> QDiagram SVG V2 n Any -> QDiagram SVG V2 n Any
 svgTitle = curry keyVal "title"
```

Comment:
How about implementing `svgId`, `svgClass`, and `svgTitle` as e.g. `svgAttr "id"` and so on?

Reply:
I'll make that change :). That was my shower thought this morning too, haha.